Test.Microsoft.NET.Build.Containers.Filesystem/EndToEnd.cs

```diff
@@ -247,7 +250,16 @@ public async Task EndToEnd_NoAPI()
             await Task.Delay(TimeSpan.FromSeconds(1));
         }
 
-        Assert.AreEqual(true, everSucceeded);
+        ProcessStartInfo logsPsi = new("docker", $"logs {appContainerId}") {
```

Comment:
So this is checking "can we get the logs from our launched container?" Should we check specific output?

Reply:
This was a debug mostly to show when the test failed to help close the loop faster - no validation expected really in my mind.